File: bubble-sort/bubblesort.js
let arr = [8,5,4,10,3]
let temp = 0;
console.log("Antes del algoritmo \n" + arr)

for(let i=0; i< arr.length-1; i++){
    for(let j=0; j< arr.length-i-1; j++){
        if(arr[j] > arr [j+1]){
            temp = arr [j]
            arr[j] = arr[j+1]
            arr[j+1] = temp
        }
    }
}
console.log("Despues del algoritmo \n" +arr)

File: bubble-sort/bubblesort.py
arr = [8,5,4,10,3] 
temp = 0;
print("Antes del algoritmo \n", arr)

for i in range(len(arr)-1):
    for j in range(len(arr)-i-1):
       if(arr[j] > arr [j+1]):
            temp = arr [j]
            arr[j] = arr[j+1]
            arr[j+1] = temp

print("Despues del algoritmo \n", arr)
